jq.pyx

```diff
@@ -51,6 +60,55 @@ cdef extern from "jq.h":
     void jq_get_error_cb(jq_state *, jq_err_cb *, void **)
 
 
+cdef object _jv_to_python(jv v):
+    """Unpack a jv value into a Python object"""
+    cdef jv_kind kind = jv_get_kind(v)
+    cdef int i
+    cdef jv ik
+    cdef jv iv
```

Comment:
I try to avoid using variable names like `i`. Are there more descriptive names that could be used?

Reply:
Sure, no problem. Changed those to `idx`/`idx_key`/`idx_value`. Tell me if that's not clear enough.